server/src/test/java/com/defold/extender/ExtenderTest.java

```diff
@@ -374,13 +374,13 @@ public void testMergedContexts() throws IOException, ExtenderException {
         // TODO: Read these from the Dockerfile itself
         env.put("PLATFORMSDK_DIR", "/opt/platformsdk");
         env.put("MANIFEST_MERGE_TOOL", "/opt/local/bin/manifestmergetool.jar");
-        env.put("XCODE_12_VERSION", "12.5");
-        env.put("IOS_14_VERSION", "14.5");
+        env.put("XCODE_14_VERSION", "14.2");
+        env.put("IOS_16_VERSION", "16.2");
         env.put("LIB_TAPI_1_6_PATH", "/usr/local/tapi1.6/lib");
-        env.put("MACOS_11_VERSION", "11.3");
-        env.put("XCODE_12_CLANG_VERSION", "12.0.5");
-        env.put("SWIFT_5_VERSION", "5.0");
-        env.put("SYSROOT", "/opt/platformsdk/MacOSX11.3.sdk");
+        env.put("MACOS_13_VERSION", "13.1");
+        env.put("XCODE_14_CLANG_VERSION", "14.0.0");
+        env.put("SWIFT_5_5_VERSION", "5.5");
```

Comment:
is it swift 5.5 not 5.7? https://developer.apple.com/documentation/xcode-release-notes/xcode-14_2-release-notes

Reply:
It's a folder name. There is no "swift-5.7" in Xcode. Only "swift", "swift-5.0" and "swift-5.5"